src/deploy-cloudrun.ts

```diff
@@ -132,6 +133,7 @@ export async function run(): Promise<void> {
     if (!metadata) {
       // Set optional flags from inputs
       if (envVars) cmd.push('--update-env-vars', envVars);
+      if (secrets) cmd.push('--update-secrets', secrets);
```

Comment:
please also add logic: `installBeta = true;`

Reply:
Thanks, I missed that. Fixed in ef597d008c0783ae799d74b84d5f2a206a3d8331